openedx/features/course_search/static/course_search/templates/course_search_results.underscore

```diff
@@ -1,6 +1,5 @@
 <div class="search-info">
-    <h2 class="search-results-title"><%- gettext("Search Results") %></h2>
-    <div class="search-count"><%- totalCountMsg %></div>
+    <h2 class="search-results-title"><%- totalCountMsg %></h2>
```

Comment:
UX would like to see this bolder; can we bump the font-weight to medium?

Reply:
It was medium (500) before. I have made it 600 for ``font-weight`` settings.

Reply:
:+1: